### ISSUE TITLE
Allow importing only parent namespace

### ISSUE THREAD

**Issue by orius123** [CloudSlang/cloud-slang#429]
for example, instead of:
```yaml
imports:
  strings: io.cloudslang.base.strings
  print: io.cloudslang.base.print
  cmd: io.cloudslang.base.cmd
  utils: io.cloudslang.base.utils
...
  <some task>
  cmd.run_command:
  <other task>
  print.print_text:
```

```yaml
imports:
  base: io.cloudslang.base
...
  <some task>
  base.cmd.run_command:
  <other task>
  base.print.print_text:
```

**Comment by Bonczidai** [comment]
I think we can close this issue since it was solved in #446

**Comment by orius123** [comment]
Agree
